scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -214,10 +215,15 @@ const processRelatedAssetIds = async (
     return
   }
 
-  const relatedAssetsResult = await getRelatedAssetIds(assetId, assetData).catch(e => {
-    console.error(`Error fetching related assets for ${assetId}: ${e}`)
-    return undefined
-  })
+  const relatedAssetsResult = await getRelatedAssetIds(assetId, assetData)
+    .then(result => {
+      console.log(`Fetched related assets for ${assetId}`)
```

Comment:
Improves logging by console logging happy ones too so we can get logs on whether 
- all are happy
- all are sad (meaning some upstream issue or rate limiting, if we see 429s or 500 for *all*) 
- or only *some* are sad (which is actually expected for some assets which will 500 upstream)

Reply:
this looks like it would be very verbose, we could also just leverage a success/fail count and print summary at the end for vis

Reply:
Definitely verbose but doesn't hurt IMO. Agree on the success/fail count idea though, what about adding it as a follow-up since there will be a few more PRs coming up for this to generate the rest and improve the flow?

Reply:
Keeping this open to reference the follow-up in a few mins

Reply:
https://github.com/shapeshift/web/pull/6589